docs/interpreter_status.md

```diff
@@ -0,0 +1,216 @@
+# StableHLO Interpreter
+
+The main goal of the StableHLO interpreter is to provide a reference
+implementation to the semantics of StableHLO opset according to its
+specification. The secondary goal is for the implementation to closely follow
+the spec to provide additional clarity to the semantics of even the most
```

Comment:
"The secondary goal is for the implementation to closely follow
the spec, favoring readability over performance, to provide additional..."

I was unclear on initial read what this meant, but I think you meant that the impls are intended to be readable?